ArduCopter/Parameters.h

```diff
@@ -381,12 +381,14 @@ class Parameters {
         k_param_vehicle = 257, // vehicle common block of parameters
         k_param_throw_altitude_min,
         k_param_throw_altitude_max,
-
+        k_param_land_accel_limit,       // 260
```

Comment:
should get rid of comment...default is not 200 anymore...

Reply:
Sure fixed that. Thanks